pkcs11/pkcs11.go

```diff
@@ -1210,8 +1250,10 @@ var hashPrefixes = map[crypto.Hash][]byte{
 }
 
 type rsaPrivateKey struct {
-	o   Object
-	pub *rsa.PublicKey
+	o    Object
+	pub  *rsa.PublicKey
+	hash *crypto.Hash
```

Comment:
yeah, per comments below, let's drop the hash and CK_OBJECT_HANDLE from this struct